### PR TITLE
#841 Removed the 'Leftover Budget' field from the Stage Gate Modal

### PR DESCRIPTION
**Pull request by walker-sean**
## Changes

The field asking about leftover budget and all associated code has been removed.

## Notes

The form will still submit correctly sans budget information.

## Screenshots

Screen Shot 2023-03-13 at 7 37 44 PM

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)
#841

### CODE REVIEW

File: src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal.tsx [hunk 1]
```diff
@@ -124,5 +97,4 @@ const StageGateWorkPackageModal: React.FC<StageGateWorkPackageModalProps> = ({ w
     </Dialog>
   );
 };
-
```
Comment (callmejim1226) [inline src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal.tsx:127]: For code formatting, make sure to have line space, including the above lines (lines 5, 25, 42) where you deleted. In other words, restore the deleted lines.

**Review by callmejim1226 — COMMENTED** [review]
Great job! The application was running fine as the ticket was intended on my local machine. 

Other than comment on the code formatting, update the tests that were failing by deleting them (two expect lines).

File: src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal.tsx (hunk 1 above)
Comment (walker-sean) [reply src/frontend/src/pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal.tsx:127]: On it!

**Review by anthonybernardi — APPROVED** [review]
awesome!

**Pull request merged**